src/test/java/org/gitlab/api/GitlabUploadTest.java

```diff
@@ -0,0 +1,60 @@
+package org.gitlab.api;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import org.apache.commons.io.IOUtils;
+import org.gitlab.api.models.GitlabProject;
+import org.gitlab.api.models.GitlabUpload;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+@Ignore
+public class GitlabUploadTest {
+
+	private static final String TEST_URL = System.getProperty("TEST_URL", "http://localhost");
```

Comment:
There is still some inconsistent formatting in various files.

Are you using a particular IDE?